### ISSUE TITLE
Can't login in Desktop app

### ISSUE THREAD

**Issue by Origena** [streetwriters/notesnook#5821]
### What happened?

I can't login using the same password that allow me to log in on the mobile app or the browser app.
I get a wrong password message and there are no options to do anything else.
Uninstalled and reinstalled - does not ask for my email.

### Steps to reproduce the problem

launch the desktop app
enter password

### Version

latest as of today

### Platform/OS

Windows

**Comment by thecodrr** [comment]
What you are seeing is probably not your account password but your app lock password. Both of those are independent. You cannot replace one with the other.

**Comment by thecodrr** [comment]
If you have really forgotten your app lock password, you'll have to delete the %APPDATA%\Notesnook directory. Do note that deleting this directory will delete everything you have stored locally so if you haven't synced something, it'll be deleted forever.

**Comment by Origena** [comment]
Thx. I did a deep clean, and I was able to access.
May i suggest rewording the prompt so that it is clearer that it is the App Lock and not the Password that is required? Or to hint that this is not the password.
Also, does that mean that using the process above, anyone could access the app (should they have the login and password?)
R
